README.md

```diff
@@ -727,7 +727,8 @@ header: # <1>
   license:
     spdx-id: Apache-2.0 # <2>
     copyright-owner: Apache Software Foundation # <3>
-    content: | # <4>
+    name: skywalking-eyes # <4>
```

Comment:
@kezhenxu94 mentioned about concerns about (name). Recommend (softwareName).

Reply:
Oh, I missed it.